src/common/form/Form.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import * as React from 'react';
+import { Provider, observer } from 'mobx-react';
+import FormStore from './FormStore';
+
+@observer
+class Form extends React.Component {
+  formStore: FormStore;
```

Comment:
nit: `_formStore` since it should be private.

Reply:
ok, but just since it's a `react component`, we usually don't separate inner state to private/public.

Reply:
acutally, no.. I push it to Provider to be able to get it inside FormField component, so I won't rename it.